docs/pages/blog/material-ui-v6-sneak-peek.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Material UI v6 Sneak Peek
+description: Previewing all the major updates of Material UI v6.
+date: 2024-07-01T00:00:00.000Z
+authors: ['siriwatknp']
+tags: ['Material UI', 'Product']
+manualCard: true
+---
+
+Material UI v6 is now in beta! We're excited to share a sneak peek of all the upcoming major updates.
+
+## Opt-in CSS extraction via Pigment CSS
+
+As you know, Material UI v5 is using Emotion as a default styling solution. As a runtime CSS-in-JS library, it has several trade-offs such as slower performance and larger bundle size.
+
+With Material UI v6, we're introducing an opt-in integration with Pigment CSS, our new open-source library, that will eliminate the overhead of style recalculation while preserving similar APIs that you are already familiar with.
+
+Since the integration is an opt-in feature, you can upgrade to v6 and continue using Emotion if you'd like. We will provide a migration guide to help you switch to Pigment CSS at your pace.
+
+<!-- would be nice if we have a link to a prototype page -->
+<!-- a link to Next.js repo that render templates -->
+
+### React Server Components
+
+Once integrated with Pigment CSS, Material UI v6 provides a separate set of layout components, including `Grid`, `Container`, and `Stack`, that are compatible with React Server Components.
+
+```jsx
+import Grid from '@mui/material/PigmentGrid';
+import Container from '@mui/material/PigmentContainer';
+import Stack from '@mui/material/PigmentStack';
+```
+
+<!-- A link to example repo? -->
+
+## CSS theme variables
+
+A new API that lets you turn your theme into CSS variables.
```

Comment:
Should we rephrase this to say that the experimental `CssVarsProvider` has been stabilized instead of saying "a new api"?